eng/Versions.props

```diff
@@ -130,21 +128,24 @@
     <SystemPrivateUriPackageVersion>4.3.2</SystemPrivateUriPackageVersion>
     <VS_NewtonsoftJsonPackageVersion>12.0.2</VS_NewtonsoftJsonPackageVersion>
     <VSMAC_NewtonsoftJsonPackageVersion>12.0.2</VSMAC_NewtonsoftJsonPackageVersion>
-    <StreamJsonRpcPackageVersion>2.2.53</StreamJsonRpcPackageVersion>
-    <VSIX_MicrosoftVisualStudioTelemetryPackageVersion>16.0.4-master</VSIX_MicrosoftVisualStudioTelemetryPackageVersion>
-    <VSIX_MicrosoftCodeAnalysisAnalyzersPackageVersion>2.9.8</VSIX_MicrosoftCodeAnalysisAnalyzersPackageVersion>
-    <VSIX_MicrosoftCodeAnalysisPackageVersion>3.4.0</VSIX_MicrosoftCodeAnalysisPackageVersion>
-    <VSIX_MicrosoftCodeAnalysisCommonPackageVersion>3.4.0</VSIX_MicrosoftCodeAnalysisCommonPackageVersion>
-    <VSIX_MicrosoftCodeAnalysisCSharpFeaturesPackageVersion>3.4.0</VSIX_MicrosoftCodeAnalysisCSharpFeaturesPackageVersion>
-    <VSIX_MicrosoftCodeAnalysisCSharpPackageVersion>3.4.0</VSIX_MicrosoftCodeAnalysisCSharpPackageVersion>
-    <VSIX_MicrosoftCodeAnalysisCSharpWorkspacesPackageVersion>3.4.0</VSIX_MicrosoftCodeAnalysisCSharpWorkspacesPackageVersion>
-    <VSIX_MicrosoftCodeAnalysisEditorFeaturesTextPackageVersion>3.4.0</VSIX_MicrosoftCodeAnalysisEditorFeaturesTextPackageVersion>
-    <VSIX_MicrosoftCodeAnalysisFeaturesPackageVersion>3.4.0</VSIX_MicrosoftCodeAnalysisFeaturesPackageVersion>
-    <VSIX_MicrosoftCodeAnalysisRemoteRazorServiceHubPackageVersion>3.4.0-beta4-19606-05</VSIX_MicrosoftCodeAnalysisRemoteRazorServiceHubPackageVersion>
-    <VSIX_MicrosoftCodeAnalysisVisualBasicWorkspacesPackageVersion>3.4.0</VSIX_MicrosoftCodeAnalysisVisualBasicWorkspacesPackageVersion>
-    <VSIX_MicrosoftCodeAnalysisWorkspacesCommonPackageVersion>3.4.0</VSIX_MicrosoftCodeAnalysisWorkspacesCommonPackageVersion>
-    <VSIX_MicrosoftVisualStudioLanguageServicesPackageVersion>3.4.0</VSIX_MicrosoftVisualStudioLanguageServicesPackageVersion>
-    <VSIX_MicrosoftVisualStudioLanguageServicesRazorRemoteClientPackageVersion>3.4.0-beta4-19606-05</VSIX_MicrosoftVisualStudioLanguageServicesRazorRemoteClientPackageVersion>
+    <Runtime_MicrosoftCodeAnalysisCommonPackageVersion>3.4.0</Runtime_MicrosoftCodeAnalysisCommonPackageVersion>
+    <Runtime_MicrosoftCodeAnalysisCSharpPackageVersion>3.4.0</Runtime_MicrosoftCodeAnalysisCSharpPackageVersion>
+    <StreamJsonRpcPackageVersion>2.3.103</StreamJsonRpcPackageVersion>
+    <Tooling_MicrosoftCodeAnalysisWorkspacesCommonPackageVersion>3.6.0-3.20168.4</Tooling_MicrosoftCodeAnalysisWorkspacesCommonPackageVersion>
```

Comment:
@KirillOsenkov would upping our tooling dependency to this cause issues in VS4Mac?

Reply:
Don't think so but @sandyarmstrong to verify.

Reply:
Potentially, but if so, only for a short while. VSmac is stuck on `3.6.0-2.20164.4` until https://github.com/dotnet/roslyn/issues/42606 is fixed.

Given that we're not too far behind it's likely this won't cause any problems (and we can just wait to insert Razor until after the Roslyn issue is resolved).